src/App/components/InputGroup/index.js

```diff
@@ -42,7 +42,9 @@ const InputGroup = ({
     errorMessage,
     successMessage,
     tooltip,
-    optional
+    optional,
+    postfix,
+    boxSize
```

Comment:
What is this used for? Can't see it being used anywhere 🤔

Reply:
This is for making the actual input-field (the box) size smaller or medium sized. When we eventually will implement Filter-page we will need a slimmer version of the inputfield (as a search field). Therefore the styling is already finalized.